src/jobs/backfill/backfill-transaction-block-fields.ts

```diff
@@ -17,8 +17,8 @@ export const queue = new Queue(QUEUE_NAME, {
   connection: redis.duplicate(),
   defaultJobOptions: {
     attempts: 10,
-    removeOnComplete: 10000,
-    removeOnFail: 10000,
+    removeOnComplete: 100,
```

Comment:
just to confirm: how `removeOnComplete: 100` is different than `removeOnComplete: true`

Reply:
it allow 100 jobs to stay

Reply:
is there any benefit to keep 100 jobs (already completed) in queue?